roles/docker/tasks/systemcontainer_crio.yml

```diff
@@ -104,8 +105,9 @@
 
     - name: Use RHEL based image when distribution is Red Hat
       set_fact:
-        l_crio_image_prepend: "registry.access.redhat.com"
+        l_crio_image_prepend: "registry.access.redhat.com/openshift3"
         l_crio_image_name: "cri-o"
+        l_docker_image_tag: "{{ openshift_image_tag }}"
       when: ansible_distribution == "RedHat"
```

Comment:
Anyone running RHEL and installing origin would be pulling CRI-O from the openshift3 namespace, I believe we should check for `openshift_deployment_type == "openshift-enterprise"` instead.